src/test/java/util/SupportUtils.java

```diff
@@ -128,6 +140,26 @@ public static boolean processCsvAndDump(byte[] bytes) throws IOException {
         return processor.parseRecords();
     }
 
+    /**
+     * Given a byte[] bytes, attempt to parse those bytes as CSV, transforming each record into
+     * JSON, separately posting that JSON to postUri.
+     * @param bytes
+     * @param postUri
+     * @return
+     * @throws IOException
+     */
+    public static boolean processCsvAndPost(byte[] bytes, String postUri) throws IOException {
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            String fakeRequestId = "" + System.currentTimeMillis();
```

Comment:
Is this variable needed? If so, could the name get changed please?

Reply:
The request id is needed to be fed into the survey sender but I've changed the name